packages/cli/src/databases/sqlite/migrations/index.ts

```diff
@@ -24,6 +25,7 @@ const sqliteMigrations = [
 	UpdateWorkflowCredentials1630330987096,
 	AddExecutionEntityIndexes1644421939510,
 	CreateUserManagement1646992772331,
+	LowerCaseEmail1648740597343,
```

Comment:
Obviously out of scope, but is there anything against reading all migrations from the dir instead of having to remember to add each one manually to the index every time for every DB?

Reply:
nothing that comes up to my mind atm, let's add it to tech debt?